index.js

```diff
@@ -830,13 +838,14 @@
                       reprs.join(_(', ')));
 
     var padding = _(showTypeSig_(expTypes.slice(0, index)));
```

Comment:
I believe this line should be something like

`var padding = _(showTypeSig_(expTypes.slice(0, isNaN(index) ? expTypes.length - 1 : index)));`

to properly point to type variables in the return value.